### PR TITLE
feat: stretch input box for dashboard search to fill space horizontally

### PR DESCRIPTION
**Pull request by Chandru-HM**
# Description
Stretch input box for dashboard search to fill space horizontally in the home page #1286 
https://ibb.co/TWFStzD

## Legal

This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx [hunk 1]
```diff
@@ -244,6 +244,7 @@ export function DashboardsIndexPage() {
                 },
                 { dashboardCount: filteredItemsCount },
               )}
+              className="dashboard-table-filter" // no props available for this in cloudscape to show full width
```
Comment (mnischay) [inline apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx:247]: classname is deprecated or will be deprecated i think ??? 
if so, any other way to override the width ?

Comment (Chandru-HM) [reply apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx:247]: yeah, classname is deprecated.
other easy way is wrapping the PropertyFilter inside div and that it'll be like classname > div > div.

**Comment by diehbria** [comment]
please update target branch to `rc` 🙏

File: apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx (hunk 1 above)
Comment (mnischay) [reply apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx:247]: can you update the CR with the above approach ?

**Review by mnischay — CHANGES_REQUESTED** [review]

**Review by mnischay — COMMENTED** [review]
can you also please rebase to the lastest rc, not sure why the checks arent running

File: apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx (hunk 1 above)
Comment (Chandru-HM) [reply apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx]: updated.

**Comment by Chandru-HM** [comment]
pr branch is upto date with rc branch.

**Pull request closed without merging**